### PR TITLE
Update --debug to --inspect

### PR DESCRIPTION
**Pull request by patrickreynolds**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "dev-start": "source ./env/development && source ./env/dev-secrets && node --debug app.js",
+    "dev-start": "source ./env/development && source ./env/dev-secrets && node --inspect app.js",
```
Comment (albertalquisola) [inline package.json:9]: bah i tried to update this too but it wasnt working with my dev tools natively :/.  Might need more investigation before we change this (maybe i need to update chrome)

Comment (patrickreynolds) [reply package.json:9]: Wasn't working on mine either. Wanted to start a PR so we're tracking it as a todo

Comment (patrickreynolds) [reply package.json:9]: Non-urgent though

Comment (albertalquisola) [reply package.json:9]: good idea.  yeah, would love for this to just work so we can scrap node-inspector which has been very buggy for me in the past. (it works now though so gotta make due)

**Comment by albertalquisola** [comment]
donzo here:
f3c704f2b1083716c37878ce5aec6a31d6fc1113

**Pull request closed without merging**